app/src/components/containers/AttributeListFormField/attributeListFormField.jsx

```diff
@@ -59,8 +59,8 @@ export const AttributeListFormField = ({
       ? attributesNote || formatMessage(messages.attributesNote)
       : formatMessage(messages.attributesNotActive);
 
-  const attributeControlHandler = (e) => {
-    setShowEditor(e.target.checked);
+  const attributeControlHandler = (isChecked) => {
+    setShowEditor(isChecked);
```

Comment:
I'm looking for some best practices here and mostly see that component libraries try to follow the native DOM API as much as possible (see f.e. [material UI checkbox](https://mui.com/material-ui/api/checkbox), another example is a React Synthetic events), so I suppose in this case it's better to pass an `event` object to the callback.
Not only to be consistent with browser APIs, but this also can be useful for some event related properties (e.g. `stopPropagation`, `target`, etc.)